pylint/checkers/dunder_methods.py

```diff
@@ -17,20 +18,22 @@ class DunderCallChecker(BaseChecker):
     """Check for unnecessary dunder method calls.
 
     Docs: https://docs.python.org/3/reference/datamodel.html#basic-customization
-    We exclude __init__, __new__, __subclasses__, __init_subclass__,
-    __set_name__, __class_getitem__, __missing__, __exit__, __await__,
-    __del__, __aexit__, __getnewargs_ex__, __getnewargs__, __getstate__,
+    We exclude __new__, __subclasses__, __init_subclass__, __set_name__,
+    __class_getitem__, __missing__, __exit__, __await__,
+    __aexit__, __getnewargs_ex__, __getnewargs__, __getstate__,
     __setstate__, __reduce__, __reduce_ex__
     since these either have no alternative method of being called or
     have a genuine use case for being called manually.
 
-    Additionally we exclude dunder method calls on super() since
-    these can't be written in an alternative manner.
+    Additionally we exclude uninstantiated classes since these
+    might be used to access the dunder methods of a base class of an instance.
     """
 
     __implements__ = IAstroidChecker
 
     includedict = {
+        "__init__": "Instatiate class directly",
+        "__del__": "Use del keyword",
         "__repr__": "Use repr built-in function",
```

Comment:
We were only excluding these before since these were being used within dunder method defs. Now that we exclude dunder calls within dunder method defs in general, we can flag these bad uses as well 👍